test/test_view/test_fileproxy.py

```diff
@@ -271,211 +271,211 @@ def test_proxy_key_not_found(self, create_connection):
+
+#     @with_context
+#     @patch('pybossa.view.fileproxy.HDFSKerberos.get')
+#     @patch('pybossa.view.fileproxy.requests.get')
+#     def test_proxy_key_err(self, http_get, hdfs_get):
+#         res = MagicMock()
+#         res.json.return_value = {'error': 'an error occurred'}
+#         http_get.return_value = res
+
+#         admin, owner = UserFactory.create_batch(2)
+#         project = ProjectFactory.create(owner=owner, info={
+#             'ext_config': {
+#                 'encryption': {'key_id': 123}
+#             }
+#         })
+#         url = '/fileproxy/hdfs/test/%s/file.pdf' % project.id
+#         task = TaskFactory.create(project=project, info={
+#             'url': url
+#         })
+
+#         signature = signer.dumps({'task_id': task.id})
+#         req_url = '%s?api_key=%s&task-signature=%s' % (url, admin.api_key, signature)
+
+#         with patch.dict(self.flask_app.config, self.app_config):
+#             res = self.app.get(req_url, follow_redirects=True)
+#             assert res.status_code == 500, res.status_code
+
+#     @with_context
+#     @patch('pybossa.view.fileproxy.HDFSKerberos.get')
+#     @patch('pybossa.view.fileproxy.requests.get')
+#     def test_offset_length(self, http_get, hdfs_get):
+#         res = MagicMock()
+#         res.json.return_value = {'key': 'testkey'}
+#         http_get.return_value = res
+
+#         project = ProjectFactory.create(info={
+#             'ext_config': {
+#                 'encryption': {'key_id': 123}
+#             }
+#         })
+#         url = '/fileproxy/hdfs/test/%s/file.ndjson?offset=10&length=10' % project.id
+#         task = TaskFactory.create(project=project, info={
+#             'url': url
+#         })
+#         owner = project.owner
+
+#         signature = signer.dumps({'task_id': task.id})
+#         req_url = '%s&api_key=%s&task-signature=%s' % (url, owner.api_key, signature)
+
+#         encryption_key = 'testkey'
+#         aes = AESWithGCM(encryption_key)
+#         hdfs_get.return_value = aes.encrypt('the content')
+
+#         with patch.dict(self.flask_app.config, self.app_config):
+#             res = self.app.get(req_url, follow_redirects=True)
+#             assert res.status_code == 200, res.status_code
+#             assert res.data == b'the content', res.data
+
+
+# def test_is_file_url():
+#     file_url = '/a/b'
+#     qps = {'offset': ['1'], 'length': ['2']}
+#     is_valid_url = is_valid_hdfs_url(file_url, qps)
+#     assert not is_valid_url(1)
+#     assert not is_valid_url('/a/c')
+#     assert not is_valid_url('/a/b?offset=1')
+#     assert not is_valid_url('/a/b?offset=1&offset=2')
+#     assert not is_valid_url('/a/b?length=2')
+#     assert not is_valid_url('/a/b?offset=1&length=3')
+#     assert not is_valid_url('/a/b?offset=1&length=3&task-signature=asdfasdfad')
+#     assert is_valid_url('/a/b?offset=1&length=2')
+#     assert is_valid_url('/a/b?offset=1&length=2&task-signature=asdfasdfas')
```

Comment:
Instead of commenting out these deprecated code blocks, it would be better to delete them entirely.

Reply:
Yes, after grace period from we fully deprecate, we can remove commented code; keeping it for reference purpose for some time.